langstream-core/src/main/java/ai/langstream/impl/common/BasicClusterRuntime.java

```diff
@@ -354,6 +354,7 @@ private Topic buildImplicitTopicForDeadletterQueue(
                 new TopicDefinition(
                         name,
                         creationMode,
+                        TopicDefinition.CREATE_MODE_NONE,
```

Comment:
this should be the same as the original topic, the same way we do for "creationMode"